doc/sphinx/language/core/variants.rst

```diff
@@ -64,6 +67,18 @@ be defined using :cmd:`Variant`.
       Variant option (A : Type) : Type := None : option A | Some : A -> option A.
       Variant sum (A B : Type) : Type := inl : A -> sum A B | inr : B -> sum A B.
 
+.. example::
+
+  :cmd:`Variant` cannot be used to define recursive types.
```

Comment:
```suggestion
.. example:: :cmd:`Variant` won't define recursive types
```

This will show the text in the header, like this:
![image](https://github.com/user-attachments/assets/46719916-8fb8-4b35-bf9c-8c851d7c39df)